test/acceptance/features/steps/steps.py

```diff
@@ -84,6 +84,7 @@ def sbo_is_not_running_in_namespace(context, operator_namespace):
     Checks if the SBO is not running in the given namespace
     """
     sb_operator = Servicebindingoperator(namespace=operator_namespace)
+    sb_operator.uninstall()
```

Comment:
If that's a `Check`, it may not be desired to have it changing the state by uninstalling SBO.

Do you think it may be a good idea to add a new step for `uninstall` or to rename this one to have it explicitly saying that it will delete SBO and check if it has been correctly uninstalled?